opentelemetry-instrumentation/README.rst

```diff
@@ -11,11 +11,14 @@ Installation
 
 ::
 
-    pip install opentelemetry-instrumentation
+    pip install opentelemetry-distro[otlp] opentelemetry-instrumentation
```

Comment:
I think we should separate out these sections. Right underneath this, an explanation like what you have in the note would be good (users need to install "A" distro), and then have the `pip install` follow that excerpt. An explanation that `opentelemetry-distro` contains the default distro would be good too. This way users won't be confused as to why or what the `opentelemetry-distro` packge is.